### ISSUE TITLE
Test for unhandled promise rejection tracking

### ISSUE THREAD

**Issue by keirlawson** [compat-table/compat-table#745]
As per https://github.com/tc39/ecma262/pull/76

**Comment by ljharb** [comment]
fwiw, that tracking is not part of the language spec. It's something browsers _can_ choose to opt in to - and those hooks you linked to are just so that the browsers don't have to write overriding spec text.

I don't think it's something we should be testing for, because it's not required by ES - but nor is it something we _can_ test for in any implementation I've seen (unhandled rejection tracking is console-only)

**Comment by zloirock** [comment]
I disagree. [Promise rejection tracking events](https://github.com/whatwg/html/pull/224) can be added as a part of a new table for ES-related HTML features, like #636.

**Comment by ljharb** [comment]
Sure, if it's in a new table that makes sense. The tests shouldn't be added until there's at least one browser implementation, though.

**Comment by zloirock** [comment]
It's implemented in Chrome 49+.

**Comment by ljharb** [comment]
Good to know - once that ships stable then my conditions are met :-)

**Comment by zloirock** [comment]
Strange condition. Why do you think so? :) It's a spec.

**Comment by Daniel15** [comment]
Since Chrome supports this now, I'll see if I can add it to the table.

**Comment by zloirock** [comment]
To which table? :) We still haven't a table for it.

**Comment by Daniel15** [comment]
Haha. That's a good point. I guess it doesn't make sense on the ES6 page since it's specific to browsers. ¯\_(ツ)_/¯

In any case, `'onunhandledrejection' in window` seems to be a sufficient test for it. Looks like only Chrome supports it right now.

**Comment by kangax** [comment]
What about non-standard table?

**Comment by zloirock** [comment]
@kangax it's already a part of HTML standard. As I wrote before, makes sense add a new table for ES-related HTML features.

**Comment by chicoxyzzy** [comment]
IMO this should be on caniuse website instead. or we'll end up creating a copy of caniuse in a new tab of compat-table
